runtime/help/options.md

```diff
@@ -386,6 +382,25 @@ Here are the available options:
 
     default value: `2`
 
+* `showchars`: sets what characters to be shown to display various invisible
+   characters in the file. The characters shown will not be inserted into files.
+   This option is specified in the form of `key1=value1,key2=value2,...`.
+
+   Here are the list of keys:
+   - `space`: space characters
+   - `tab`: tab characters, this option overrides the `indentchar` option
```

Comment:
"this key"?
Or, cleaner, in a separate sentence: "If set, overrides the `indentchar` option."?